### DISCUSSION TITLE
Add basic electrical parameters to the readme

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#7] [Ideas]
While assembling the whole system you have to search for values in datasheets and calculate other values.
It would be helpful to have basic parameters in readme such as (but not limited to):

- Maximum current draw of the whole module during heavy network load
- Maximum switched voltage
- Maximum switched current
- Min/Max input signal voltage

**Comment by pilotak** [comment]
perfect idea, V3 has a little different values https://github.com/pilotak/eth-io/blob/28b9ccb64864791d090ab6205445d9660984af67/stickers/ETH-IO_v3_stickers.pdf

**Reply by mDubovsky** [reply]
Very nice work with the sticker!

**Reply by pilotak** [reply]
Thank you but the design is heavily inspired by yours so credits goes to you :sunglasses:

**Comment by pilotak** [comment]
I would set max continuous input voltage to 30VDC (it can do AC too, but you would need to adjust the FW and it's considered as safe voltage too).
It can do more but would heat up a bit, or you can replace series resistors and set it higher to let say 64V. Higher than that would require bigger package due to small spacing resistor array. If you want to go 230AC you would have to put multiple resistors is series.

Minimal really depends on CTR of the optocoupler as it start at minimum 50% or 100% for GB version. I calculated it once in other project, but in real scenario it was completely different so i gave up on that. In order to find out i would have to measure it.

**Reply by mDubovsky** [reply]
I would choose something in a "safe ballpark". Even if lower/higher values are possible the values given should be only a guidance for people that don't want to experiment a lot and just build it the way it works. People that would want to modify the circuit to an extreme (such as 230V input voltage) are probably (hopefully) experienced enough to figure out which components need to be changed and how so I wouldn't really consider leaving a manual for them :)

**Reply by pilotak** [reply]
you are right, i will measuse it later
